centreon/src/Centreon/Infrastructure/Contact/ContactRepositoryRDB.php

```diff
@@ -390,6 +390,31 @@ private function parseLocaleFromContactLang(string $lang): string
         return $locale;
     }
 
+    /**
+     * Get the default timezone.
+     *
+     * @return string
+     *   The timezone name.
+     */
+    private function getDefaultTimezone(): string
+    {
+        $query = <<<SQL
+                SELECT timezone.timezone_name
+                FROM `:db`.timezone
+                JOIN `:db`.options ON timezone.timezone_id = options.value
+                WHERE options.key = 'gmt'
+                SQL;
+
+        $query = $this->translateDbName($query);
+
+        $statement = $this->db->prepare($query);
+        $statement->execute();
+
+        $result = $statement->fetch(\PDO::FETCH_ASSOC);
+
+        return $result['timezone_name'] ?? date_default_timezone_get();
+    }
```

Comment:
Code style please 🙏🏻 
```suggestion
    /**
     * Get the default timezone.
     *
     * @return string The timezone name
     */
    private function getDefaultTimezone(): string
    {
        $query = <<<'SQL'
            SELECT timezone.timezone_name
            FROM `:db`.timezone
            JOIN `:db`.options ON timezone.timezone_id = options.value
            WHERE options.key = 'gmt'
            SQL;

        $query = $this->translateDbName($query);

        $statement = $this->db->prepare($query);
        $statement->execute();

        $result = $statement->fetch(\PDO::FETCH_ASSOC);

        return $result['timezone_name'] ?? date_default_timezone_get();
    }
```